inline.h

```diff
@@ -3042,6 +3042,8 @@ Perl_foldEQ_locale(const char *s1, const char *s2, I32 len)
 {
     const U8 *a = (const U8 *)s1;
     const U8 *b = (const U8 *)s2;
+    dTHX;   /* XXX pTHX_ for this, but would have to make all similar fcns the
```

Comment:
IMO, `dTHX` is generally a bad idea. This really should be using `pTHX_` instead.